examples/simple/README.md

```diff
@@ -0,0 +1,6 @@
+
+# Simple Trace Example
+
+In this example, the application in `main.cc` initializes and registers a tracer provider from the [OpenTelemetry SDK](https://github.com/open-telemetry/opentelemetry-cpp).  The application then calls a `foo_library` which has been instrumented using the [OpenTelemetry API](https://github.com/open-telemetry/opentelemetry-cpp/tree/master/api).  Resulting telemetry is directed to stdout through a custom exporter.
```

Comment:
Would you add line wrapping at column 80?

Reply:
Use one space instead of two for consistency.

Reply:
fixed both :)